.circleci/config.yml

```diff
@@ -43,6 +44,18 @@ jobs:
           command: |
             echo "export TAG=ci-build-$CIRCLE_BRANCH" >> $BASH_ENV
       - *build_and_push
+  build_1804_deb:
```

Comment:
not sure why the deb suffix is for, this should be generic image/build

Reply:
It is not tested for general building, only for Ubuntu/Debian package building. Probably it should work. Additional tools are added. I got the impression it will be used only for the package building. At least in the foreseeable future.

Reply:
You can build a normal rebar package as well, that what I mean by "general"

Reply:
So, it's general builder container used to build all package variants based on ubuntu 18. For example if we add appimage support we still should build it using this container

Reply:
Fixed in d30a4bc